modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -226,6 +226,12 @@ HTTP messages not containing this header field will be rejected. Two common HTTP
 *Default value*: Blank and unchecked.
 
 |=== 
++
+If you are using WebSockets:
++
+* Provide the correct request headers to upgrade the HTTP connection to WebSockets.
+* xref:mule-runtime::mule-server-notifications[Configure Mule runtime engine with a WebSockets Listener].
+* Increase the `Connection Idle Time-out` value to 900 seconds (15 minutes) to ensure consistency with the WebSockets Mule application default value.
```

Comment:
"WebSockets Mule application default value" is a noun string of five nouns. Could you revise it to something like this? "the default time-out value of the Mule application for WebSockets"
"the default time-out value of the WebSockets Mule application" (if "WebSockets" is the name of the Mule application)